edumap/config/environments/development.rb

```diff
@@ -38,4 +38,15 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
+  config.action_mailer.smtp_settings = {
```

Comment:
Yep I put up a warning for it, see: config/environments/shared.rb. Email send is always disabled for test mode.